src/wallet/wallet.cpp

```diff
@@ -3238,7 +3240,9 @@ bool CWallet::SelectPSInOutPairsByDenominations(int nDenom, CAmount nValueMin, C
         return false;
     }
 
-    AvailableCoins(vCoins, true, nullptr, 1, MAX_MONEY, MAX_MONEY, 0, 0, 9999999, ONLY_DENOMINATED);
+    CCoinControl coin_control;
+    coin_control.nCoinType = CoinType::ONLY_DENOMINATED;
```

Comment:
Why do we now sometimes set nCoinType directly to ONLY_DENOMINATED and sometimes use `UsePrivateSend()`?

Reply:
I assigned coin type directly here to keep it unified with of other use cases in `wallet.cpp`. I implemented it with Is/UsePrivateSend helpers first actually but it felt a bit weird to use a mix of direct assignments and helpers, so I decided to use helpers in qt only as a replacement for `fUsePrivateSend` (which mirrors the state of the "PrivateSend" checkbox).